test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CertignaCA.java

```diff
@@ -23,70 +23,13 @@
 
 /*
  * @test
- * @bug 8245654 8314960
+ * @bug 8245654 8256895
```

Comment:
The tests for `Certigna Root CA` is moved to `CAInterop` but `Certigna` CA test will continue to use old framework as there are no test URLs.

This test is also update test for JDK-8256895 which was part of SSLCA.java test